fulp_modules/main_features/species/beefmen/mob/beefman.dm

```diff
@@ -722,3 +723,34 @@
 /datum/sprite_accessory/beef/mouth/smile2
 	name = "Smile2"
 	icon_state = "smile2"
+
+/obj/item/organ/tongue/beefman_tongue
+	name = "Meaty sausage tongue"
```

Comment:
```suggestion
	name = meaty sausage tongue"
```

Reply:
are U just TRYING to be an ASSHOLE mane becoz if u are then ur SUcceeding